backend/apps/sandbox/views.py

```
@@ -1,32 +1,69 @@
     command = serializers.CharField()
     expected_command = serializers.CharField()
+    # Optional fields for duplicate prevention
+    code = serializers.CharField(required=False, default='')
+    payload = serializers.JSONField(required=False, default={})
 
 
 class SandboxVerifyView(APIView):
+    """
+    Sandbox verification with duplicate execution prevention.
+    """
     permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)